default/stringtables/en.txt

```diff
@@ -1891,6 +1891,9 @@ Enable autosave when resigning or closing the game.
 OPTIONS_DB_AUTOSAVE_HOSTLESS
 Enable autosave in hostless mode.
 
+OPTIONS_DB_AUTOSAVE_TIMEOUT
+Duration in seconds when game will be saved after turn started. Prevents losing player orders on server crash. 0 if disabled.
```

Comment:
Delay from turn start until the single autosave occurs, or the delay after the most recent turn start or autosave until the next autosave?

Reply:
Fixed.